File: React/src/types/model/ElementInWarehouse.ts
export type ElementInWarehouse = {
    id: number
    inWarehouseCount: number
    inUnitCount: number
    rack: string
    shelf: string
    elementId: number
    warehouseId: number
}

export type ElementInWarehouseFilterDto = {
    id: number
    inWarehouseCount: number
    inUnitCount: number
    rack: string
    shelf: string
    element: string
    warehouse: string
}

File: Android/app/src/main/java/com/example/e_montazysta/ui/element/ElementInWarehouse.kt
package com.example.e_montazysta.ui.element

data class ElementInWarehouse(
    val deleted: Boolean,
    val elementId: Int,
    val id: Int,
    val inUnitCount: Int,
    val inWarehouseCount: Int,
    val rack: String,
    val shelf: String,
    val warehouseId: Int
)